code/game/objects/structures/tables_racks.dm

```diff
@@ -858,12 +858,12 @@
 	deconstruct(TRUE)
 	return ITEM_INTERACT_SUCCESS
 
-/obj/structure/rack/item_interaction(mob/living/user, obj/item/tool, list/modifiers, is_right_clicking)
-	. = ..()
-	if(. || (tool.item_flags & ABSTRACT) || user.combat_mode)
-		return .
+/obj/structure/rack/item_interaction(mob/living/user, obj/item/tool, list/modifiers)
+	if((tool.item_flags & ABSTRACT) || user.combat_mode)
+		return NONE
 	if(user.transferItemToLoc(tool, drop_location(), silent = FALSE))
 		return ITEM_INTERACT_SUCCESS
+	return NONE
```

Comment:
Does this mean we'll hit it if transfer fails? Can this be made into a guard instead?

Reply:
I suppose we can
```suggestion
	if(user.transferItemToLoc(tool, drop_location(), silent = FALSE))
		return ITEM_INTERACT_SUCCESS
	return ITEM_INTERACT_BLOCKING
```